### PR TITLE
Consolidate order price fetching to a single method

### PR DESCRIPTION
**Pull request by CydeWeys**

### CODE REVIEW

**Comment by gbrodman** [comment]
Does this solve the issue you were seeing? If that issue was caused by calling get_order_info multiple times for the same tracking number, wouldn't it still happen here?

**Comment by CydeWeys** [comment]
This just removes the duplicate code and adds a progress bar for the second call.

I can't say whether the 2nd call is even necessary at all; is it?

File: reconcile.py [hunk 1]
```diff
@@ -15,24 +15,7 @@
...
-        print(
-            f"Exception when getting order info for {order_id}. Please check the oldest email associated with that order. Skipping..."
-        )
-        print(str(e))
-        continue
-      cluster.expected_cost += order_info.cost
-
-
-def fill_email_ids(all_clusters, config):
+def fill_costs(all_clusters, config, also_fill_email_ids: bool = False):
```
[the remark this answers is not in the document]
Comment (CydeWeys) [reply reconcile.py:18]: So rename this option `replace_email_ids`?

[the remark this answers is not in the document]
Comment (gbrodman) [reply reconcile.py:18]: sure that'd work

**Comment by CydeWeys** [comment]
Ended up going a different direction with it.  It turns out it doesn't matter whether or not we populate the email ids since we only merge on them once, so I simplified that bit.

**Pull request merged**